packages/eslint-plugin/configs/jsdoc.js

```diff
@@ -1,5 +1,32 @@
 const globals = require( 'globals' );
 
+/**
+ * Helpful utilities that are globally defined and known to the TypeScript compiler.
+ *
+ * @see http://www.typescriptlang.org/docs/handbook/utility-types.html
+ */
+const typescriptUtilityTypes = [
```

Comment:
Ideally, this comes from the official TypeScript npm package but I guess it wouldn't be as easy to extract. I did a quick google search and I didn't find anything that would fit.

Reply:
Yeah, unfortunately they aren't able to be extracted in this way because they're defined as `type`s and/or `interface`s. It would be much cleaner though if that was possible, I agree.